### PR TITLE
Update managing logs page in Lisk docs

### LINKED ISSUE #1533: Update `Managing logs`
### Description

- Update `Managing logs` to use Lisk SDK v6 interface.

### Location
run-blockchain/logging.adoc

### PR DESCRIPTION
**Pull request by chris529**
Closes #1533

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 1]
```diff
@@ -7,7 +7,8 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_bunyan: https://github.com/trentm/node-bunyan
 :url_bunyan_docs: http://trentm.com/node-bunyan/bunyan.1.html
 
-For monitoring or debugging your node, the Lisk SDK tracks all activities that occur within the node by creating applicable log messages.
+
+For monitoring or debugging your node, the Lisk SDK v6 tracks all activities that occur within the node by creating applicable log messages.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc:11]: Please check, if the numbers (10-60) are still used to identify log levels, or if they can be removed.

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 2]
```diff
@@ -34,18 +35,35 @@ This also requires the operator to investigate the problem, although it does not
 |Trace(10) |Logging from external libraries used by your node, or otherwise very detailed application logging.
 |===
 
-== Logging destinations
+By using the following config setting listed in the snippet below, it is possible to set the log level of the app.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc]:
```suggestion
To change the node log level, update the `logLevel` property in the `config.json` file, under xref:v6@lisk-sdk::config.adoc#system[system.logLevel] as listed in the snippet below.
```

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 3]
```diff
@@ -34,18 +35,35 @@ This also requires the operator to investigate the problem, although it does not
...
+version: '0.1.0',
+keepEventsForHeights: 300,
+logLevel: 'info',
+enableMetrics: false,
+backup: {
+height: 0,
+},
+}
+}
+----
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc]: Please add indentation.

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 4]
```diff
@@ -34,18 +35,35 @@ This also requires the operator to investigate the problem, although it does not
 |Trace(10) |Logging from external libraries used by your node, or otherwise very detailed application logging.
 |===
 
-== Logging destinations
+By using the following config setting listed in the snippet below, it is possible to set the log level of the app.
+
+[source,bash]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc]:
```suggestion

.`system` options in config.json
[source,json]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please add the new page to the navigation as well.

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc (hunk 1 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/logging.adoc:11]: Yes, they are still valid and correct 👍

**Review by Incede — APPROVED** [review]

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 5]
```diff
@@ -54,67 +72,29 @@ The default log level for the console log stream is `none`.
...
-
-For example, Ubuntu systems  provide a service called `logrotate` specifically for this purpose.
-Please ensure Logrotate is installed on your system as shown below:
-
-[source,bash]
+All log file support has been removed from the Lisk SDK as the application `stdout output` can be simply output to a file by a user as required.
+For example, on Ubuntu and MacOS you can already use the CLI options to output to a file as shown below.
 ----
-logrotate --version
+./bin/run start  > app.log will output stdout to a file app.log
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc]: the part after app.log seems to be a comment
```suggestion
./bin/run start  > app.log # will output stdout to a file app.log
```

File: docs/modules/ROOT/pages/run-blockchain/logging.adoc [hunk 6]
```diff
@@ -54,67 +72,29 @@ The default log level for the console log stream is `none`.
...
 ----
-
-Now go to the logrotate config directory and create a new logrotate file for Lisk SDK as shown below:
-
-[source,bash]
+To continue on appending in the same file run the following command.
 ----
-cd /etc/logrotate.d
-touch lisk
+./bin/run start  >> app.log
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/logging.adoc]: Looks like the same command as above, can be removed.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good already, just left two small remarks :)

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**